### ISSUE TITLE
Issue #14 — Train Baseline CNN

### ISSUE THREAD

**Issue by christine-a11y** [christine-a11y/cassava-leaf-disease-classification#15]
Նպատակ

Training անել baseline model-ը։

Tasks
 Սահմանել loss function
 Սահմանել optimizer
 Սահմանել learning rate
 Սահմանել batch size
 Սահմանել epochs
 Train անել model-ը
 Պահել training history

Հետևել
Training loss
Validation loss
Training accuracy
Validation accuracy
Deliverable

Trained baseline model + training history։
